src/main/java/com/company/elixr/springbootapplicationwithdocker/model/FileInfo.java

```diff
@@ -0,0 +1,27 @@
+package com.company.elixr.springbootapplicationwithdocker.model;
+
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.FieldType;
+import org.springframework.data.mongodb.core.mapping.MongoId;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.GeneratedValue;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Document(collection = "files")
+@Data
+@Component
+public class FileInfo {
+
+    @MongoId(FieldType.STRING)
```

Comment:
Why is the type given as string?

Reply:
I have removed the "FieldType.String" part